code/modules/admin/verbs/spatialagent.dm

```diff
@@ -0,0 +1,48 @@
+/client/verb/spatialagent()
+	set category = "Debug"
+	set name = "Spawn Spatial Agent"
+	set desc = "Spawns a Spatial Agent. Useful for debugging."
+
+	if (!check_rights(R_DEBUG))
+		return
+
+	var/T = get_turf(usr)
+	var/mob/living/carbon/human/sa/sa = new(T)
+	sa.ckey = usr.ckey
+	sa.name = "Spatial Agent"
+	sa.real_name = "Spatial Agent"
+	sa.voice_name = "Spatial Agent"
+	sa.h_style = "Cut Hair"
+
+	var/datum/job/J = new /datum/job/other/spatial_agent
+	J.generate_equipment(sa)
+
+	//Languages.
+	sa.add_language("English")
```

Comment:
Add this to the job datum's generate_entry_conditions, call that above like you did with generate_equipment.